### PR TITLE
[V2] Fix multiple checkboxes

### PR DESCRIPTION
**Pull request by rvsia**

### CODE REVIEW

**Comment by Hyperkid123** [comment]
:skull_and_crossbones:

File: packages/common/src/multiple-choice-list.js [hunk 1]
```diff
@@ -1,70 +1,52 @@
...
+      {options.map((option) => (
+        <SingleCheckbox
+          Checkbox={Checkbox}
+          aria-label={option['aria-label'] || option.label}
+          {...rest}
+          value={option.value}
+          label={option.label}
+          name={input.name}
+          option={option}
+          id={`${rest.id || input.name}-${option.value}`}
```
Comment (Hyperkid123) [inline packages/common/src/multiple-choice-list.js]: The `rest.id` won't work on its own. All checkboxes will have the same id. Either add the -option.value or we should remove the id entirely and just use name/option combination.

Comment (rvsia) [reply packages/common/src/multiple-choice-list.js]: there is a `-${option.value}`

**Pull request merged**